app/src/components/MessageList.tsx

```diff
@@ -221,6 +263,9 @@ class MessageList extends Component<Props, State> {
             <div className={classes.scrollContainer}>
                 <Box className={classes.tableHeaderRow}>
                     <Box className={classNames(classes.tableColumn, classes.overrideTimestampColumn)}>
+                        <CustomTooltip title={'Click the Tilde ` to manually center to selected message'}>
```

Comment:
"Click tilde ` to center to selected message"

Reply:
👍